api/lib/opentelemetry/baggage/propagation/text_map_propagator.rb

```diff
@@ -51,7 +51,7 @@ def inject(carrier, context: Context.current, setter: Context::Propagation.text_
         #
         # @return [Context] context updated with extracted baggage, or the original context
         #   if extraction fails
-        def extract(carrier, context: Context.current, getter: Context::Propagation.text_map_getter)
+        def extract(carrier, context: Context.current, getter: Context::Propagation.text_map_getter) # rubocop:disable Metrics/AbcSize
```

Comment:
should we disable this rule altogether?

Reply:
I would personally like to disable that, yes. But I do not hold this view strongly.

Reply:
I'm down to disable globally...will handle in separate PR

Reply:
https://github.com/open-telemetry/opentelemetry-ruby/pull/1329